docs/getting-started/setup/macos.md

```diff
@@ -112,9 +112,9 @@ Before running the above command, it's recommended you review the script content
 Then, install the JDK with:
 
 ```shell
-sdk install java 17.0.10-tem
-sdk use java 17.0.10-tem
-sdk default java 17.0.10-tem
+sdk install java 21.0.03-tem
+sdk use java 21.0.03-tem
+sdk default java 21.0.03-tem
```

Comment:
There is a `21.0.3-tem` version but no `.03`. There's also an umbrella `21-tem` option for SDKMAN.